core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProvider.scala

```diff
@@ -108,8 +159,8 @@ class CompactEntityProvider(requestArguments: EntityRequestArguments, repository
         _ <- replaceReferences(savedEntityRecord.id, referencedIds.toSet, isInsert = true)
       } yield savedEntityRecord.toEntity
     }
-    // fire-and-forget an update to the workspace's last-modified date; no need to wait for it to complete
     withWorkspaceLastModified(createFuture)
+    // fire-and-forget an update to the workspace's last-modified date; no need to wait for it to complete
```

Comment:
Xtreme nit: shouldn't this comment stay above the `withWorkspaceLastModified` line?